Lib/configparser.py

```diff
@@ -692,6 +692,10 @@ def add_section(self, section):
         if section == self.default_section:
             raise ValueError('Invalid section name: %r' % section)
 
+        if section is UNNAMED_SECTION:
+            if not self._allow_unnamed_section:
+                raise ValueError("Unnamed section not enabled")
```

Comment:
```suggestion
        if section is UNNAMED_SECTION and not self._allow_unnamed_section:
	   		raise ValueError("Support for UNNAMED_SECTION is disabled")
```

Do you perhaps want add a specific exception?

Reply:
Yes, just added.